modules/javafx.graphics/src/main/native-glass/mac/GlassApplication.m

```diff
@@ -204,6 +187,7 @@ - (void)applicationWillFinishLaunching:(NSNotification *)aNotification
     NSAutoreleasePool *pool = [[NSAutoreleasePool alloc] init];
     {
         // unblock main thread. Glass is started at this point.
+        self->platformSupport = [[PlatformSupport alloc] initWithEnv:env application:jApplication];
```

Comment:
should we check for failure here?  (alloc)

Reply:
We never do that in obj-c native code, so this probably wouldn't be the place to start. In any case, a failure at this point wouldn't lead to a crash, as the only access of `platformSupport` checks for `nil`.